src/index.html

```diff
@@ -17,6 +17,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Logo"
+        />
+      </a>
```

Comment:
Add empty lines between multiline sibling blocks of HTML. There should be an empty line between the `a` tag with class 'logo' and the `nav` tag.